tidb-configuration-file.md

```diff
@@ -362,6 +362,8 @@ Configuration items related to performance.
 - Default value: `6291456` (in bytes)
```

Comment:
In L361, do we need to mention this configuration item also controls the size limit of a single column?

Reply:
I don't think that's needed as this should be implicit as columns can't be bigger than the row.